lib/api_client.ex

```diff
@@ -13,10 +11,10 @@ defmodule GlassFactoryApi.ApiClient do
   Make a GET request using the given adapter to retrieve the given resource
 
   ## Examples
-
       iex> GlassFactoryApi.ApiClient.get("members")
       {:ok , %{body: "[]", headers: [], status_code: 200}}
   """
+
   @spec get(String.t()) :: http_response()
   def get(resource, adapter \\ GlassFactoryApi.HTTPoisonAdapter) do
```

Comment:
Tesla has a [good list of adapters](https://github.com/teamon/tesla#adapters). Adopting Tesla in this project can be a good experiment on how to make it flexible for the library users. For example, if my project uses Gun to make HTTP requests, I can continue using it when I add `glass_factory` to my dependencies.

Reply:
I looked at their docs and it looks like a very nice project! I'm creating an issue to adopt Tesla over HTTPoison! 👍 😄